D2L.Security.OAuth2.sln

```diff
@@ -17,9 +17,11 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
 		Directory.Build.props = Directory.Build.props
-		Packages.props = Packages.props
+		Directory.Packages.props = Directory.Packages.props
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "D2L.Security.OAuth2.Benchmarks", "test\D2L.Security.OAuth2.Benchmarks\D2L.Security.OAuth2.Benchmarks.csproj", "{E24E112F-AA04-4685-AE76-3127074719EF}"
+EndProject

```

Comment:
Was inadvertently removed in https://github.com/Brightspace/D2L.Security.OAuth2/pull/365